### ISSUE TITLE
Support Discord Secrets

### ISSUE THREAD

**Issue by ysknsid25** [secretlint/secretlint#1010]
## Summary

GitHub Secret Scannning support discord_bot_token string.
refs: https://docs.github.com/en/enterprise-cloud@latest/code-security/secret-scanning/introduction/supported-secret-scanning-patterns

It may also support Discord Webhook as well as Slack.

### Basic example

**token**

maybe

![Image](https://github.com/user-attachments/assets/6d694df4-faa9-41ea-9dba-c7fb27d84902)

**Webhook**

start with `https://discord.com/api/webhooks/*`

### Motivation

It is common to be notified of spam by including the Discord Webhook or token in public repositories.
Also, if Slack is already supported, it would be good to support Discord as well.
Secretlint should support it.

**Comment by azu** [comment]
Intersting.

Since Timestamp is big-endian, it seems that a little ingenuity is needed in decoding.

- https://ru.stackoverflow.com/questions/1200650/%D0%9A%D0%BE%D0%BD%D0%B2%D0%B5%D1%80%D1%82%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D1%82%D1%8C-big-int-number-%D0%B2-base64

```js
base64BigEndianToNumber("DN9r_A"); // 215968764

function base64BigEndianToNumber(s) {
    // base64 url to base64
    const base64 = s.replace(/-/g, '+').replace(/_/g, '/');
    // add padding
    const paddedBase64 = base64.padEnd(base64.length + (4 - base64.length % 4) % 4, '=');
    
    // Node.js
    if (typeof Buffer !== 'undefined') {
        const bytes = Buffer.from(paddedBase64, 'base64');
        return bytes.readUInt32BE(0);
    } else {
        // Browser - TODO more effeciant
        const binary = atob(paddedBase64);
        const binaryArrayBuffer = new DataView(new ArrayBuffer(binary.length));
        for (let i = 0; i < binary.length; i++) {
            binaryArrayBuffer.setUint8(i, binary.charCodeAt(i));
        }
        // read BigEndian
        return binaryArrayBuffer.getUint32(0);
    }
}
```

Since UserId and TimeStamp are only numbers, you can use a more specific regular expression for Base64 encoding, such as `^[0-9]+$` to match only digits.

**Comment by ysknsid25** [comment]
I have some time this weekend, so I'll try to implement it there 👍
